docs/src/lib/functions.md

```diff
@@ -6,8 +6,15 @@ CurrentModule = DataFrames
 
 ## Multi-threading support
 
-Selected operations in DataFrames.jl automatically use multiple threads when available.
-It is task-based and implemented using the `@spawn` macro from Julia Base.
+By default, selected operations in DataFrames.jl automatically use multiple threads
+when available. It is task-based and implemented using the `@spawn` macro from Julia Base.
+Multithreading can be disabled when running a particular block of code using the
+[`DataFrames.singlethreading() do... end`(@ref) syntax. It can also be disabled
+globally by calling [`DataFrames.setmultithreading(true)`](@ref).
+This is useful in particular to run functions which are not thread-safe, or when
+distribution of work across threads is managed separately.
+These functions are considered as experimental and may change or be removed once
+a cross-package mechanism for multithreading configuration is developed.
```

Comment:
can you also add a short NEWS.md entry? (listing functions and mentioning this is experimental)